src/steadystateproblem.cpp

```diff
@@ -231,6 +288,10 @@ void SteadystateProblem::writeErrorString(std::string *errorString,
                                           SteadyStateStatus status) const {
     /* write error message according to steady state status */
     switch (status) {
+        case SteadyStateStatus::failed_too_long_simulation:
+            (*errorString).append(": Simulation beyond t=1e100 without finding "
```

Comment:
don't explicitely mention time here or extract value from variable, high chance of going stale.